jenkins/deploy.sh

```diff
@@ -53,8 +53,9 @@ fi
 
 ART_ID=`mvn help:evaluate -q -pl $DIST_PL -Dexpression=project.artifactId -DforceStdout`
 ART_VER=`mvn help:evaluate -q -pl $DIST_PL -Dexpression=project.version -DforceStdout`
+CUDA_CLASSIFIER=`mvn help:evaluate -q -pl $DIST_PL -Dexpression=cuda.version -DforceStdout`
 
-FPATH="$DIST_PL/target/$ART_ID-$ART_VER"
+FPATH="$DIST_PL/target/$ART_ID-$ART_VER-$CUDA_CLASSIFIER"
```

Comment:
As the main jar is the must have for Sonatype deploy, so we need to keep it instead of replacing it with the $CUDA_CLASSIFIER jar;  https://github.com/NVIDIA/spark-rapids/pull/5249#issuecomment-1101077865

And I'd prefer to do the same thing we do for cudf JNI jar, e.g. copy spark-rapids $CUDA_CLASSIFIER as the main jar, then deploy both the main and $CUDA_CLASSIFIER jars

Reply:
@NvTimLiu sounds good! 
I will merge this one and keep #5259 open, and yes I would prefer to do a replacement before the release (empty main jar does not look good).
Please help test and adjust the deploy script before our actual release, thanks!